src/components/viewLayout/__tests__/viewLayoutToolbar.test.tsx

```diff
@@ -2,21 +2,232 @@ import React from 'react';
 import { shallowComponent } from '../../../../config/jest.setupTests';
 import { AppToolbar as ViewToolbar } from '../viewLayoutToolbar';
 
+// Mock localStorage
+const mockLocalStorage = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  removeItem: jest.fn(),
+  clear: jest.fn()
+};
+
+// Mock window.matchMedia
+const mockMatchMedia = jest.fn();
+
+// Mock document.getElementsByTagName for theme application
+const mockGetElementsByTagName = jest.fn();
+const mockHtmlElement = {
+  classList: {
+    add: jest.fn(),
+    remove: jest.fn()
+  }
+};
+
 describe('ViewToolbar', () => {
+  beforeEach(() => {
+    // Reset mocks
+    jest.clearAllMocks();
+
+    // Setup localStorage mock
+    Object.defineProperty(window, 'localStorage', {
+      value: mockLocalStorage,
+      writable: true
+    });
+
+    // Setup matchMedia mock
+    Object.defineProperty(window, 'matchMedia', {
+      writable: true,
+      value: mockMatchMedia
+    });
+
+    // Setup document.getElementsByTagName mock
+    mockGetElementsByTagName.mockReturnValue([mockHtmlElement]);
+    Object.defineProperty(document, 'getElementsByTagName', {
+      value: mockGetElementsByTagName,
+      writable: true
+    });
+
+    // Setup default environment variable
+    process.env.REACT_APP_THEME_KEY = 'discovery-ui-theme';
```

Comment:
I'm only chiming in because I see there are conflicts that you have to resolve. And since you have to make some changes anyway...

Generally, I don't think test should set, and later unset, process environment variables. Especially as unsetting leaves the environment in **different** state than it was before running this test.

If you really want to ensure that testing environment has stable value, isolated from `.env` file changes, then that value should be added to `.env.test`.

But personally, I would just remove all `process.env` references from this test file and use whatever we have `.env`. It's internal constant, it's only affected by environment variables because **all** our constants come from environment.